layouts/downloads/list.html

```diff
@@ -15,6 +15,11 @@ <h2 >Decred Wallet For Mobile</h2>
         <a href="https://play.google.com/store/apps/details?id=com.decred.dcrandroid.mainnet" target="_blank"><img src="{{ "../images/google-store-icon.png" | absURL }}" alt="AndroidApp"></a>
         <a href="https://apps.apple.com/us/app/decred-wallet/id1462247643" target="_blank"><img src="{{ "../images/apple-store-icon.png" | absURL }}" alt="iOSAPP"></a>
     </div>
+
+    <div class="downloads-container">
+        <h2 >Decred Wallet For Computers</h2>
```

Comment:
A cross-platform desktop [SPV](https://docs.decred.org/wallets/spv/) wallet for [decred](https://decred.org/) built with [gio](https://gioui.org/)